lms/djangoapps/mobile_api/preferences/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
@gwprice I'll leave it up to the edX team to decide if you want to merge this into an existing end point - it is out of our scope at this point. 

@nasthagiri can you be more specific about the failing unit tests, I'm not sure I understand what the connection to PR you mentioned is?